pico_i2c_lcd.py

```diff
@@ -35,21 +36,25 @@ def __init__(self, i2c, i2c_addr, num_lines, num_columns):
         if num_lines > 1:
             cmd |= self.LCD_FUNCTION_2LINES
         self.hal_write_command(cmd)
+        gc.collect()
```

Comment:
gc.collect() collects garbage at the end of every function. Thanks to that the memory doesn't get eaten up